sympy/core/compatibility.py

```diff
@@ -352,8 +352,8 @@ def as_int(n, strict=True):
 
     The return value is guaranteed to be equal to the input. ValueError
     is raised if the input has a non-integral value. When ``strict`` is
-    False, non-integer input that compares equal to the integer value
-    will not raise an error.
+    False, it would call ``int`` in the ``else`` block and non-integer
```

Comment:
```suggestion
    True this uses `__index__ <https://docs.python.org/3/reference/datamodel.html#object.__index__>`_
    and when it is False it uses `int`.
```

Reply:
You should probably build the Sphinx docs to make sure that formats correctly.

Reply:
> You should probably build the Sphinx docs to make sure that formats correctly.

Great tool.. thanks @asmeurer :)